sympy/parsing/latex/lark/latex_parser.py

```diff
@@ -0,0 +1,43 @@
+import os
+import logging
+
+from sympy.external import import_module
+
+
+class LaTeXParsingError(Exception):
+    pass
+
+
+# TODO: if lark isn't found, just define a dummy transformer class, and don't do
+# any type of parsing.
+_lark = import_module('lark')
+
+if _lark is None:
+    raise ImportError("Could not load 'lark'")
+
+_lark.logger.setLevel(logging.DEBUG)
+
+
+def parse_latex_lark(s: str):
+    # TODO: should we use pkg_resource to get grammar file?  I
+    # think this would make sympy depend on setuptools which we
+    # would not like
+    with open(os.path.join(os.path.dirname(__file__), 'latex.lark')) as f:
+        latex_grammar = f.read()
+
+    parser = _lark.Lark(latex_grammar, parser='earley', start='string',
+                        lexer='auto',
+                        ambiguity='explicit',
+                        debug=True,
+                        propagate_positions=False,
+                        maybe_placeholders=False,
+                        keep_all_tokens=True)
+
+    print("expression =", s)
```

Comment:
these `print` statements should not be merged.

Reply:
As these are temporary, I'm marking this as resolved. They will be removed at the end.